benchmarking/mod/benchmark.py

```diff
@@ -248,25 +322,98 @@ def archive(self):
         #write description to dir
         with open( "%s/conf/description" % dest_dir, 'w+' ) as f:
             f.write( self.benchmark["description"] )
+
+        #copy all.conf to all node's log path
+        self.cetune_path = os.path.join(self.cluster["tmp_dir"],'CeTune/')
+        self.loacl_cetune_path = '/'
```

Comment:
so I believe this is a spell error right? I don't understand the word 'loacl', what does this word mean?